src/EA.Weee.RequestHandlers/AatfEvidence/GetTransferEvidenceNoteForSchemeRequestHandler.cs

```diff
@@ -35,12 +34,17 @@ public async Task<TransferEvidenceNoteData> HandleAsync(GetTransferEvidenceNoteF
             authorization.EnsureCanAccessExternalArea();
 
             var evidenceNote = await evidenceDataAccess.GetNoteById(request.EvidenceNoteId);
+
             var transferredScheme = await schemeDataAccess.GetSchemeOrDefaultByOrganisationId(evidenceNote.OrganisationId);
-            
+
             Condition.Requires(transferredScheme).IsNotNull();
 
-            var allowedAccess = authorization.CheckOrganisationAccess(evidenceNote.OrganisationId) || 
-                          authorization.CheckSchemeAccess(evidenceNote.RecipientId);
+            var allowedAccess = authorization.CheckOrganisationAccess(evidenceNote.OrganisationId) || authorization.CheckSchemeAccess(evidenceNote.RecipientId);
+
+            if (!allowedAccess)
+            {
+                throw new SecurityException($"The user does not have access to the organisation or scheme with note ID {request.EvidenceNoteId}");
```

Comment:
Why has this been duplicated?

Reply:
good spot - this has been doubled-up recently during a manual merge that went through without being noticed.  duplicate is removed.